test/chatterbox_test_buddy.erl

```diff
@@ -75,9 +75,8 @@ start(Config) ->
     {ok, _RanchPid} =
         ranch:start_listener(
           chatterbox_ranch_protocol,
-          10,
           ranch_ssl,
-          [{port, 8081}|proplists:get_value(ssl_options, Settings)],
+          #{ socket_opts => [{port, 8081}|proplists:get_value(ssl_options, Settings)] },
```

Comment:
`socket_opts` isn't actually required here, but this way it's more future-proof.